ui/views_test.py

```diff
@@ -561,8 +565,9 @@ def test_page_not_found(url, logged_in_apiclient, settings):
     }
 
 
-def test_terms_page(mock_moira_client, logged_in_client):
+def test_terms_page(mocker, logged_in_client):
     """Test terms page"""
+    mocker.patch('ui.utils.get_moira_client')
```

Comment:
It seems a little odd that we need the moira client for the terms of service page, but I'm fine leaving this since it's out of scope of the PR